core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -76,6 +79,12 @@ class AdminClient(val time: Time,
     response.groups().asScala.map(group => GroupOverview(group.groupId(), group.protocolType())).toList
   }
 
+  def getApiVersions(node: Node): util.Collection[ApiVersion] = {
+    val response = send(node, ApiKeys.API_VERSIONS, new ApiVersionsRequest.Builder()).asInstanceOf[ApiVersionsResponse]
+    Errors.forCode(response.errorCode()).maybeThrow()
+    response.apiVersions()
```

Comment:
Fair enough.  Let's use a scala.Try to represent the fact that we might get an exception here.